src/Uitpas/UitpasClientInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace CultuurNet\ProjectAanvraag\Uitpas;
+
+use GuzzleHttp\ClientInterface;
+
+/**
+ * Provides an interface for search clients on the uitpas api
+ */
+interface UitpasClientInterface
+{
+
+    /**
+     * Set the guzzle client.
+     *
+     * @param ClientInterface $client
+     */
+    public function setClient(ClientInterface $client);
```

Comment:
It's not common to have a `setClient` and `getClient` inside an interface if they can be changed to constructor arguments. It's always better to pass this `ClientInterface` as a constructor argument because this forces a client to be set before it is even possible to call `searchRewards`

Reply:
I took the same approach as in https://github.com/cultuurnet/projectaanvraag-silex/blob/13253a29cf39ecfa536a469974491136c8f63db2/src/Curatoren/CuratorenClientInterface.php

Could you maybe write a code suggestion on how you would do it?
Since I'm a PHP noob 😅

Reply:
The `setClient` and `getClient` methods can be removed from the interface and class implementation since the guzzle client is passed via the constructor upon creation of the UitpasClient

Reply:
@simon-debruijn thx!